rpc-rs/src/rpc_client.rs

```diff
@@ -221,21 +240,30 @@ impl RpcClient {
         &self,
         origin: WasmCloudEntity,
         target: Target,
+        lattice: &str,
         message: Message<'_>,
     ) -> RpcResult<()>
     where
         Target: Into<WasmCloudEntity>,
     {
-        let _ = self.inner_rpc(origin, target, message, false, None).await?;
-        Ok(())
+        let rc = self.inner_rpc(origin, target, lattice, message, false, None).await;
+        match rc {
+            Err(e) => {
+                #[cfg(feature = "prometheus")]
+                self.stats.rpc_sent_err.inc();
+                Err(e)
+            }
+            Ok(_) => Ok(()),
+        }
     }
 
     /// request or publish an rpc invocation
-    #[instrument(level = "debug", skip(self, origin, target, message), fields(issuer = tracing::field::Empty, origin_url = tracing::field::Empty, subject = tracing::field::Empty, target_url = tracing::field::Empty, method = tracing::field::Empty))]
+    //    #[instrument(level = "debug", skip(self, origin, target, message), fields(issuer = tracing::field::Empty, origin_url = tracing::field::Empty, subject = tracing::field::Empty, target_url = tracing::field::Empty, method = tracing::field::Empty))]
```

Comment:
Unintentional comment out?

Reply:
intentional. I think we're logging in the right places.